cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1,24 +1,269 @@
+        "#});
+
+        manifest.insert(MANIFEST_VARS_TABLE_KEY, Item::Table(vars_table));
+
+        // `[hook]` table
+        let mut hook_table = Table::new();
+
+        if let Some(ref hook_on_activate_script) = customization.hook_on_activate {
+            let on_activate_content = formatdoc! {r#"
+                """
+                {}
+                """"#
+            , indent::indent_all_by(2, hook_on_activate_script)};
+
+            hook_table.insert("on-activate", value(on_activate_content));
+        } else {
+            hook_table.decor_mut().set_suffix(indoc! {r#"
+
+                # on-activate = """
+                #     # Set variables, create files and directories
+                #     venv_dir="$(mktemp -d)"
+                #     export venv_dir
+                #
+                #     # Perform initialization steps, e.g. create a python venv
+                #     python -m venv "$venv_dir"
+                #
+                # """
+            "#});
+        };
+
+        hook_table.decor_mut().set_prefix(indoc! {r#"
+            # The `hook.on-activate` script is run by the *bash* shell immediately upon
+            # activating an environment, and will not be invoked if Flox detects that the
+            # environment has previously been activated. Variables set by the script will
+            # be inherited by `[profile]` scripts defined below. Note that any stdout
+            # generated by the script will be redirected to stderr.
+        "#});
+
+        manifest.insert(MANIFEST_HOOK_TABLE_KEY, Item::Table(hook_table));
+
+        // `[profile]` table
+        let mut profile_table = Table::new();
+
+        match customization {
+            InitCustomization {
+                profile_common: None,
+                profile_bash: None,
+                profile_zsh: None,
+                ..
+            } => {
+                profile_table.decor_mut().set_suffix(indoc! {r#"
+
+                    # common = """
+                    #     echo "it's gettin' flox in here"
+                    # """
+                    # bash = """
+                    #     source $venv_dir/bin/activate
+                    #     alias foo="echo bar"
+                    # """
+                    # zsh = """
+                    #     source $venv_dir/bin/activate
+                    #     alias foo="echo bar"
+                    # """
+                "#});
+            },
+            _ => {
+                profile_table.insert(
+                    "common",
+                    value(formatdoc! {r#"
+                        """
+                        {}
+                        """
+                    "#, value(
+                    customization
+                        .profile_common
+                        .as_deref()
+                        .map(|profile_common| indent::indent_all_by(2, profile_common))
+                        .unwrap_or("".to_string()),
+                    )}),
+                );
```

Comment:
1) `toml_edit::value` twice in this is weird (kinda calls for three sets of `"""`)
2) see above comment about adding indentation and removing `"""`